templates/pages/login.jsx

```diff
@@ -3,32 +3,38 @@ var React = require('react');
 var Form = require('../components/form.jsx');
 var Header = require('../components/header.jsx');
 
+var fieldValues = [
+  {
+    'username': {
+      'placeholder': 'Username',
+      'type': 'text',
+      'validator': 'username'
+    }
+  },
+  {
+    'password': {
+      'placeholder': 'Password',
+      'type': 'password',
+      'validator': 'password'
+    }
+  }
+];
+
+var validators = require('../lib/validatorset');
+var fieldValidators = validators.getValidatorSet(fieldValues);
+
 // This wraps every view
 var Login = React.createClass({
-
   render: function() {
     // FIXME: totally not localized yet!
     var buttonText = "Log In";
-    var fieldValues = [
-      {
-        'username': {
-          'placeholder': 'Username',
-          'type': 'text'
-        }
-      },
-      {
-        'password': {
-          'placeholder': 'Password',
-          'type': 'password'
-        }
-      }
-    ];
     return (
       <div>
-        <Header />
+        <Header redirectText="Need an account?" redirectLabel="Sign up" redirectURL="/signup" />
```

Comment:
Change
+        <Header redirectText="Need an account?" redirectLabel="Sign up" redirectURL="/signup" />

to 

+        <Header redirectText="Need an account?" redirectLabel="Sign up" redirectPage="signup" />